### DISCUSSION TITLE
[IntelliSense] addPixelEquivalents supports container query

### DISCUSSION THREAD

**Discussion by RyanYANG52** [tailwindlabs/tailwindcss#15296] [Ideas]
https://github.com/tailwindlabs/tailwindcss-intellisense/blob/805dfe18c1372048736d7752be62e68b5631ec8a/packages/tailwindcss-language-service/src/util/pixelEquivalents.ts#L57

currently it only support `rem to pixel` comment for `@media`, it would be nice to add support for 
```css
@container (min-width: 24rem /* 384px */) 
{
}
```

**Comment by christopher356** [comment]
Enjoyed reading the article above, really explains everything in detail, and the article is very interesting and effective. Thank you and good luck with the upcoming articles.  [T-Mobile Premium Handset Protection](https://www.my-tmoclaim.com/)
